cogs/utils/botdata.py

```diff
@@ -218,6 +225,13 @@ def __init__(self, botdata, guildid):
 			"type": types.TextChannel,
 			"description": "The channel in which mangobyte will post to notify about new dota patches when it detects them",
 			"example": "#dota"
+		},
+                		{
```

Comment:
This line has some spaces instead of tabs in it, could cause some issues with python i think